src/sentry/static/sentry/app/utils/theme.tsx

```diff
@@ -175,11 +175,9 @@ const button = {
     color: colors.blue,
     colorActive: colors.blue,
     background: 'transparent',
-    border: 'none',
-    borderActive: 'inherit',
-    // border: '#3d328e',
+    border: false,
+    borderActive: false,
```

Comment:
Alternatively we could use `transparent`

Reply:
This is good I think